pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -228,6 +237,7 @@ func (gb *GraphBuilder) syncMonitors(logger klog.Logger, resources map[schema.Gr
 		if monitor.stopCh != nil {
 			close(monitor.stopCh)
 		}
+		monitor.destroy()
```

Comment:
now，stopCh is used to stop Controller which get from shared.Informer().GetController(), now controller is dummy controller,  there is no effect on destory called before or after closing the stop channel.

in future,  it's better to destory before closing the stop channel? 

``` go
func (s *sharedIndexInformer) GetController() Controller {
	return &dummyController{informer: s}
}

// dummyController hides the fact that a SharedInformer is different from a dedicated one
// where a caller can `Run`.  The run method is disconnected in this case, because higher
// level logic will decide when to start the SharedInformer and related controller.
// Because returning information back is always asynchronous, the legacy callers shouldn't
// notice any change in behavior.
type dummyController struct {
	informer *sharedIndexInformer
}

func (v *dummyController) Run(stopCh <-chan struct{}) {
}

func (v *dummyController) HasSynced() bool {
	return v.informer.HasSynced()
}

func (v *dummyController) LastSyncResourceVersion() string {
	return ""
}
```